exporter/signalfxexporter/README.md

```diff
@@ -86,6 +86,10 @@ The following configuration options can also be configured:
   defined in `translation/constants.go`.
 - `timeout` (default = 10s): Amount of time to wait for a send operation to
   complete.
+- `http2_read_idle_timeout` (default = 10s): If the connection has been idle for the configured value send a ping frame for health check.
```

Comment:
```suggestion
- `http2_read_idle_timeout` (default = 10s): Send a ping frame for a health check if the connection has been idle for the configured value.
```
Nit: It might be more clear to change the order of statements here.

Reply:
thanks, fixe